### ISSUE TITLE
CMAPSS result not producing what is claimed in paper

### ISSUE THREAD

**Issue by ZeeshanAbbas** [TL-UESTC/RUL-prediction-with-Transformer#7]
Hi,
I was trying to reproduce the result, but it's not producing the result as claimed in the repo/article, not even for the pre-trained models, "best-performing models saved in folder save/final," as I run "python validation_cmapss.py --source FD001 --target FD003"  it results min RMSE = 26.0132, min score = 55565 and in article you claimed RMSE = 15.5 and score=2974, similarly for others.
I can understand there may be hyper-tuning and model convergence issues that can arise during training, but I'm a little bit worried about why the "best-performing models saved in folder save/final" don't produce the result. Can you please tell me what I'm doing wrong?
 I have attached the screen shoot of the above model evaluation result.
![Weixin Screenshot_20240201161231](https://github.com/TL-UESTC/Domain-Adaptive-Remaining-Useful-Life-Prediction-with-Transformer/assets/26795882/cbe1eb99-2d54-40ac-ae73-221ea20fc0b8)

**Comment by xinyao326** [comment]
Please make sure you are using the exact version of PyTorch and torchvision required.  Check #1.

**Comment by ZeeshanAbbas** [comment]
Thanks for your reply.

**Comment by ZeeshanAbbas** [comment]
I have other questions/ confusion; please clear my confusion.
1) Why didn't you explain the preprocessing step in the paper and code?
2) You haven't tested your model on the given test set, and it's RUL; instead, you have test on the part of the preprocessed dataset; why is it so?
 
![image](https://github.com/TL-UESTC/Domain-Adaptive-Remaining-Useful-Life-Prediction-with-Transformer/assets/26795882/3e869ecf-3c78-4eb6-a766-c3f9eeaa64ef)

**Comment by Q1abc** [comment]
I can't reproduce the result either, have you solved it?

**Comment by Q1abc** [comment]
> 嗨，我试图重现结果，但它并没有产生 repo/文章中声称的结果，即使对于预训练模型也是如此，“表现最佳的模型保存在文件夹 save/final 中”，当我运行“python validation_cmapss.py --source FD001 --target FD003”时，它的结果最小 RMSE = 26.0132，最小分数 = 55565，在文章中您声称 RMSE = 15.5 和分数 = 2974， 其他人也是如此。我可以理解在训练过程中可能会出现超调优和模型收敛问题，但我有点担心为什么“保存在文件夹保存/最终中性能最佳的模型”没有产生结果。你能告诉我我做错了什么吗？我附上了上述模型评估结果的屏幕截图。 ![Weixin Screenshot_20240201161231](https://private-user-images.githubusercontent.com/26795882/301431684-cbe1eb99-2d54-40ac-ae73-221ea20fc0b8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI2OTYyNjYsIm5iZiI6MTczMjY5NTk2NiwicGF0aCI6Ii8yNjc5NTg4Mi8zMDE0MzE2ODQtY2JlMWViOTktMmQ1NC00MGFjLWFlNzMtMjIxZWEyMGZjMGI4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTI3VDA4MjYwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE0ZmFhMTFiZjc4OGM4Nzk5NDE5MGQ2NzFjMDc2N2RmZjg5NTgwNWU3ZWU2ZjgwNmYyMTA2NjYxMTI5NTc3YzUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.5sYIKLXc7HrzXJaZ4lUYZZwQ5y7zPuUNwWDCXUur9eE)

I can't reproduce the result either, have you solved it?
